tests/KernelFlexibleCenterer_tests.py

```diff
@@ -0,0 +1,71 @@
+import unittest
```

Comment:
you don't have to name this with `_tests` in the name. It's in the tests folder, so it's implied.